addons/stock_account/tests/test_anglo_saxon_valuation_reconciliation_common.py

```diff
@@ -50,97 +117,3 @@ def _change_pickings_date(self, pickings, date):
-
-        self.output_account = self.env['account.account'].create({
-            'name': 'Test stock out',
-            'code': 'stock_account_TEST_43',
-            'user_type_id': self.env.ref('account.data_account_type_current_assets').id,
-            'reconcile': True,
-            'company_id': self.company.id,
-        })
-
-        self.valuation_account = self.env['account.account'].create({
-            'name': 'Test stock valuation',
-            'code': 'stock_account_TEST_44',
-            'user_type_id': self.env.ref('account.data_account_type_current_assets').id,
-            'reconcile': True,
-            'company_id': self.company.id,
-        })
-
-
-        self.test_product_category = self._create_product_category()
-
-        uom = self.env['uom.uom'].search([], limit=1)
-        test_product_delivery_inv_template = self.env['product.template'].create({
-            'name': 'Test product template invoiced on delivery',
-            'type': 'product',
-            'categ_id': self.test_product_category.id,
-            'uom_id': uom.id,
-            'uom_po_id': uom.id,
-        })
-        test_product_order_inv_template = self.env['product.template'].create({
-            'name': 'Test product template invoiced on order',
-            'type': 'product',
-            'categ_id': self.test_product_category.id,
-            'uom_id': uom.id,
-            'uom_po_id': uom.id,
-        })
-
-        self.test_product_order = test_product_order_inv_template.product_variant_id
-        self.test_product_order.write({
-            'name': 'The chocolate moose - order',
-            'standard_price': 42.0,
-        })
-
-        self.test_product_delivery = test_product_delivery_inv_template.product_variant_id
-        self.test_product_delivery.write({
-            'name': 'The chocolate moose - delivery',
-            'standard_price': 42.0,
-        })
-
-        self.test_partner = self.env['res.partner'].create({
-            'name': 'Ruben Rybnik',
-        })
-
-        self.product_price_unit = 66.0
-
-        # We delete the currency rate defined in demo data for USD on June 6th.
-        # This is mandatory to ensure consistency of the data generated by the test,
-        # as stock pickings created from a PO are by design always created for
-        # the current date (there is no way forcing this), meaning that they
-        # always use today's exchange rate for their valuation.
-        # Despite the fact we rewrite the date of the valuation moves artificially,
-        # we cannot correct the debit and credit values, since the anglosaxon
-        # entries get automatically reconciled (and you cannot modify a reconciled entry).
-        # So, we have to make sure that "today"'s rate will always be the last rate we
-        # created in order to ensure complete control of the test.
-        rateUSDbis = self.env.ref('base.rateUSDbis', raise_if_not_found=False)
-        if rateUSDbis:
-            rateUSDbis.unlink()
```

Comment:
yeah, well... I stubbled on that and can't find where it was replaced... Which I hope is because it's not replaced at all because we use somewhere freezegun to freeze_time and test properly the flow. 

Mmmh ? :pray: